playpen/ansible/roles/dev/tasks/main.yml

```diff
@@ -17,9 +17,17 @@
       dest: /etc/yum.repos.d/fedora-pulp.repo
 
 - name: Enable Pulp 2.7 beta repository
-  command: yum-config-manager --enable pulp-2.7-beta
+  command: dnf config-manager --set-enabled pulp-2.7-beta
   when: pulp_27_beta_repo_enabled == false
 
+# This and its related fact should go away when we do have an f23 build, since it'll be a noop at that point
+- name: Rejigger the repo file if fedora 23 builds aren't available
+  replace:
+      dest: /etc/yum.repos.d/fedora-pulp.repo
+      regexp: 'fedora-\$releasever'
+      replace: fedora-22
+  when: not pulp_27_beta_f23_repo_available and ansible_distribution == 'Fedora' and ansible_distribution_version == '23'
```

Comment:
I think this is a bit silly, but I assume the fedora-pulp repos for fedora 23 should be available soon. :)